<ui version="4.0" >
 <class>backgroundMain</class>
 <widget class="QDialog" name="backgroundMain" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>206</red>
        <green>207</green>
        <blue>210</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>206</red>
        <green>207</green>
        <blue>210</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>206</red>
        <green>207</green>
        <blue>210</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>206</red>
        <green>207</green>
        <blue>210</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Additional Information Needed - XSupplicant</string>
  </property>
  <property name="windowIcon" >
   <iconset>icons/prod_color.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="whiteFrameMain" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>480</width>
     <height>10</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>480</width>
     <height>300</height>
    </size>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>127</red>
         <green>157</green>
         <blue>185</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>127</red>
         <green>157</green>
         <blue>185</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>119</red>
         <green>119</green>
         <blue>119</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth" >
    <number>1</number>
   </property>
   <widget class="QFrame" name="dataFrameVersionInformation" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>460</width>
      <height>230</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>460</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize" >
     <size>
      <width>460</width>
      <height>2048</height>
     </size>
    </property>
    <property name="palette" >
     <palette>
      <active>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>208</red>
          <green>208</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>236</red>
          <green>234</green>
          <blue>223</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>208</red>
          <green>208</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>236</red>
          <green>234</green>
          <blue>223</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>119</red>
          <green>119</green>
          <blue>119</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>236</red>
          <green>234</green>
          <blue>223</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>236</red>
          <green>234</green>
          <blue>223</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground" >
     <bool>true</bool>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>190</y>
       <width>461</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>186</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOK" >
        <property name="minimumSize" >
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>186</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="dataFieldYourResponse" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>157</y>
       <width>320</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize" >
      <size>
       <width>320</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>320</width>
       <height>25</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="maxLength" >
      <number>255</number>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelYourResponse" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>160</y>
       <width>115</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize" >
      <size>
       <width>115</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>115</width>
       <height>18</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Your Response :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelServerRequest" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>66</y>
       <width>115</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize" >
      <size>
       <width>115</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>115</width>
       <height>18</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Server Request :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAdditionalInformation" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>38</y>
       <width>436</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>18</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>The server has requested additional information.</string>
     </property>
    </widget>
    <widget class="QLabel" name="headerInfoNeeded" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>18</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>144</red>
           <green>141</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>144</red>
           <green>141</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <property name="text" >
      <string>Additional Information</string>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="indent" >
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="dataFieldServerRequest" >
     <property name="geometry" >
      <rect>
       <x>129</x>
       <y>63</y>
       <width>320</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>320</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>320</width>
       <height>90</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>If you can see this, something bad happened!</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
